### DISCUSSION TITLE
Figma (Color) Variable support.

### DISCUSSION THREAD

**Discussion by NicolasBrunnerSRF** [RedMadRobot/figma-export#220] [Ideas]
We use figma-export happily since many years. Thanks for making and supporting it :)

Our design team is rapidly switching to the new [Figma variable](https://help.figma.com/hc/en-us/articles/15339657135383-Guide-to-variables-in-Figma) (currently beta) feature and ideally we'd like to export them directly to our XCode/Android Studio projects.

Do you have any plans of adding variable functionality to the exporter?

**Comment by subdan** [comment]
Hello. I currently don't have any plans to add variable functionality to the exporter.

**Reply by NicolasBrunnerSRF** [reply]
Hello and thanks for the fast response!
